src/templates/app/user/tnc_approval.jsx

```diff
@@ -8,15 +8,14 @@ const TncApproval = () => (
 
             <div className='gr-row gr-parent invisible' id='tnc_approval'>
                 <div className='gr-2 gr-6-m gr-centered-m'>
-                    <p><img className='responsive' src={it.url_for('images/pages/cashier/protection-icon.svg')} /></p>
+                    <img className='responsive' src={it.url_for('images/pages/cashier/protection-icon.svg')} />
                 </div>
                 <div className='gr-10 gr-12-m'>
+                    <h2>{it.L('We have updated our terms and conditions')}</h2>
                     <form>
-                        <p id='tnc_message'>{it.L('[_1] has updated its <a href="[_2]">Terms & Conditions</a>. By clicking OK, you confirm that you have read and accepted the updated <a href="[_2]">Terms & Conditions</a>.', '[_1]', '[_2]')}</p>
-                        <div className='gr-3 gr-6-m gr-centered'>
-                            <span className='button'>
-                                <button className='button' id='btn_accept'>{it.L('OK')}</button>
-                            </span>
+                        <p id='tnc_message'>{it.L('As part of our rebranding, we have updated our <a href="[_2]">terms and conditions</a> which can be viewed at Deriv.com. Note that all [_1] users are now bound by the terms and conditions of Deriv. Please review and accept them to continue using our services. ', it.website_name, '[_2]')}</p>
```

Comment:
Do we use this id (`tnc_message`) here?

Reply:
Yes in here:
"src/javascript/app/pages/user/tnc_approval.js"